src/load_inputs/load_generators_data.jl

```diff
@@ -501,4 +502,21 @@ function load_vre_stor_data!(inputs_gen::Dict, setup::Dict, path::AbstractString
 		inputs_gen["dfVRE_STOR"] = DataFrame()
 	end
 	summarize_errors(error_strings)
-end
\ No newline at end of file
+end
+
+
+function process_piecewisefuelusage!(inputs_gen::Dict,gen_in::DataFrame, scale_factor)
```

Comment:
I don't think there's a good reason to pass both `inputs_gen` and `gen_in` since
`inputs_gen["dfGen"]` is exactly the same as `gen_in`.

In general, `inputs_gen` should be renamed to `inputs` within this function since it's the same object.